pkg/tsdb/grafana_ds/grafana.go

```diff
@@ -0,0 +1,230 @@
+package grafana_ds
```

Comment:
If not named `grafana` then this should be `grafanads` to align with convention. I thought `grafana` was fine though as it lives under `tsdb`. It goes in line with all the other plugins here 🤷‍♂️

Reply:
nads!   I would like to avoid `grafana.Service` -- I'll change this to `grafanads` and call it a day :)